src/users/dto/create-user.dto.ts

```diff
@@ -9,4 +9,8 @@ export class CreateUserDto {
   age: number;
   @IsBoolean()
   isStudent: boolean;
+  @IsString()
+  password: string;
+  accessToken?: string;
+  refreshToken?: string;
 }
```

Comment:
fr this config it's better to have 
@IsOptional() for optional checks
to not to leave it empty